langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -248,7 +248,7 @@ void should_extract_custom_POJO() {
                         "\"firstName\": (type: string),\n" +
                         "\"lastName\": (type: string),\n" +
                         "\"birthDate\": (type: date string (2023-12-31)),\n" +
-                        "\"address\": (type: {\n" +
+                        "\"address\": (type: dev.langchain4j.service.AiServicesIT$Address: {\n" +
```

Comment:
would be nicer to have just `Address` instead of `dev.langchain4j.service.AiServicesIT$Address`

Reply:
I had three thought on that, the first I think is that it's the classname format that was already in use, the second is that package name may contain domain term that can be useful for llm understanding (or not), the third is the most important is for classname uniqueness.
So I stayed with the format already in use but if you think it's better, let's go with classname without packages names.

Reply:
Good point, thanks for explanation!